apps/stable_diffusion/src/utils/resources/opt_flags.json

```diff
@@ -29,15 +29,21 @@
           "cuda": [],
           "default_device": [
             "--iree-preprocessing-pass-pipeline=builtin.module(func.func(iree-flow-detach-elementwise-from-named-ops,iree-flow-convert-1x1-filter-conv2d-to-matmul,iree-preprocessing-convert-conv2d-to-img2col,iree-preprocessing-pad-linalg-ops{pad-size=32},iree-linalg-ext-convert-conv2d-to-winograd))"
+          ],
+          "default_lowmem": [
+            "--iree-preprocessing-pass-pipeline=builtin.module(func.func(iree-flow-detach-elementwise-from-named-ops,iree-flow-convert-1x1-filter-conv2d-to-matmul,iree-preprocessing-pad-linalg-ops{pad-size=32}))"
```

Comment:
Did you miss `iree-linalg-ext-convert-conv2d-to-winograd` flag? Or did you drop this intentionally?